core/src/main/scala/chisel3/Printf.scala

```diff
@@ -100,6 +121,14 @@ object printf {
     printfId
   }
 
+  def _applyWithReset(pable: Printable)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): Printf = {
+    var printfId: Printf = null
+    when(!Module.reset.asBool) {
+      printfId = printfWithoutReset(pable)
+    }
+    printfId
```

Comment:
Why is this here? This feels like an unrelated change.

Reply:
I've moved this around so that this logic is part of the function `printfWIthReset` so that i can call it from the macro